types/matter-js/package.json

```
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/matter-js",
-    "version": "0.19.9999",
+    "version": "0.20.0",
```

Should have patch of `.9999` see https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#packagejson

```suggestion
    "version": "0.20.999",
```